src/Application/AnalyzeCommand.php

```diff
@@ -291,7 +291,9 @@ public function analyze(
             if (is_a($data->collecterType, TemplateContextCollector::class, true)) {
                 foreach ($data->data as $renderData) {
                     $template = $this->twigFileNormalizer->normalize($renderData['template']);
-                    $templateToRenderPoint[$template][$data->filePath][] = $renderData['startLine'];
+                    $templatePath = $this->twigFileNormalizer->toAbsolute($template);
```

Comment:
Not sure about this. The goal of the `normalize` method is to handle absolute and relative file paths, and convert them to Twig style references: `@__main__/My/File.twig`. When a file is already in this format, it should stay in that format.

Maybe we should start with a proper unit test for this, and make sure we tackle all the cases?